### PR TITLE
Handle support request

### PR DESCRIPTION
**Pull request by gmarcos87**
This adds an endpoint to record support requests. Also generates a rsa key pair and sends the public one to the person who requested support.

It also adds two methods to establish remote connection.

```npm run keys:list -a [api-key]```
to list the keys and support requests 

```npm run keys:list -a [api-key] -c [private key index number]```
to connect remotely using a stored key.

### CODE REVIEW

File: scripts/listKeys.js [hunk 1]
```diff
@@ -3,26 +3,126 @@ const db = require('../src/db')
...
+                            let skipNext = false;
+                            stream.stdout.on("data", (data) => {
+                                if (skipNext) { return skipNext = false; }
+                                process.stdout.write(data);
+                            })        
+                            process.stdin.on("data", stdinListener)
+                        });
+                    })
+                    .connect({
+                        host: "mattermost.altermundi.net", //data.keys[connect].ip,
```
Comment (germanferrero) [inline scripts/listKeys.js]: We can remove the hardcode now?

**Review by germanferrero — COMMENTED** [review]
It would be nice to have all test passing successfully

File: scripts/listKeys.js (hunk 1 above)
Comment (gmarcos87) [reply scripts/listKeys.js]: My bad, I already edited the commit.

**Comment by gmarcos87** [comment]
> It would be nice to have all test passing successfully

Done!

**Review by germanferrero — APPROVED** [review]

**Comment by germanferrero** [comment]
@gmarcos87 I can't merge in this repo.

**Pull request merged**